### PR TITLE
Add Japan sings

### PR DESCRIPTION
**Pull request by chezzdev**
Checks:

* [x] Update changelog
* [x] Rebase to `dev` branch
* [x] Assign reviewers

Linked issues:
Adjust to https://github.com/mapbox/mapbox-vision/issues/2313

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.0-32
+
+- Add japanese traffic signs
```
Comment (dersim-davaod) [inline CHANGELOG.md:5]: nit:

```suggestion
- Added japanese traffic signs
```

as we use past simple earlier?

File: demo/Signs.swift [hunk 2]
```diff
@@ -1354,6 +1362,14 @@ extension Sign {
                 return nil
             case .speedLimitEndZone:
                 return nil
+            case .informationRestrictedParking:
```
Comment (dersim-davaod) [inline demo/Signs.swift:1365]: nit: Should we have the same order as for

https://github.com/mapbox/vision-sdk-trainer/blob/2826e98e245d8d42c2658874ed9e47b26fe4d4a3/trainers/classifier/export/export_utils.py#L214

for convenience?

Comment (dersim-davaod) [reply demo/Signs.swift:1365]: nevermind :)

File: CHANGELOG.md (hunk 1 above)
Comment (chezzdev) [reply CHANGELOG.md]: Right, thanks.

File: demo/Signs.swift (hunk 2 above)
Comment (chezzdev) [reply demo/Signs.swift:1365]: Hm, I can see the same order.

**Pull request merged**